readme.md

```diff
@@ -0,0 +1,225 @@
+
+Simulation tasks of this kind must be solved in the RF industry frequently because the transition needs to be designed for each specific layer stack with its own characteristic substrate materials, thicknesses and metal layers.
+
+A list of open source software used in this project:
+- [OpenEMS](https://www.openems.de/)
+- [FreeCAD](https://www.freecad.org/)
+- [GNU Octave](https://octave.org/)
+- [KiCAD](https://www.kicad.org/)
+- [ParaView](https://www.paraview.org/)
+- [QUCS](https://qucs.sourceforge.net/)
+- [VS Code](https://code.visualstudio.com/)
+- [Inkscape](https://inkscape.org/)
+- [IEEE P370 Standard](https://opensource.ieee.org/elec-char/ieee-370)
+
+## 1. Table of Contents
+
+- [1. Table of Contents](#1-table-of-contents)
+- [2. Rosenberger 32K242-40ML5 SMA Connector](#2-rosenberger-32k242-40ml5-sma-connector)
+- [3. The Connector Model](#3-the-connector-model)
+- [4. PCB Design Considerations and Manufacturing](#4-pcb-design-considerations-and-manufacturing)
+- [5. Simulation Workflow](#5-simulation-workflow)
+- [6. Simulation Results](#6-simulation-results)
+- [7. Measurement Results](#7-measurement-results)
+- [8. Conclusion](#8-conclusion)
+
+
+## 2. Rosenberger 32K242-40ML5 SMA Connector
+I have chosen a typical right angle jack SMA connector. This connector type requires a cutout in the PCB edge which is an additional design and manufacturing effort.
+
+<p align = "center"><img src="./figures/edge_mount_connector.jpg" alt="fig_conn_wuerth" width="500"></p>
+<p align = "center">Figure 1: A typical edge mount SMA connector to be mounted in a cutout</p>
+
+However, I like these connectors because the solder mount is very strong due to the large mounting area which results in very good mechanical durability. The parasitic inductance in the return current is also low because the solder joint is right at the circular edge of the coaxial outer conductor.
+
+ Connectors of this kind are available from multiple manufacturers (Molex, Würth, Rosenberger...). I chose the one which was cheapest for me to source which was the one manufactured by Rosenberger. Therefore the 32K242-40ML5 made the run, purchased for about 6€ at rs-online.com. Figure 1 shows an exemplary CAD model of the Würth 60312202114307 (Rosenberger does not provide CAD a model for their part).
+
+## 3. The Connector Model
+Typically the first step in designing a coaxial to microstrip transition is to request a simulation or mechanical model from the connector vendor. 
+
+Today, most vendors provide detailed models for commercial simulators. However, in most cases a NDA (Non Disclosure Agreement) is required . Sometimes the internals of the connector are even encrypted and therefore not visible to the user (e.g. SV Microwave).
+
+I had no hope that Rosenberger would send a model to me - a private person without a NDA. I tried it nevertheless - with no success. They did not even bother to respond to my email.
```

Comment:
```suggestion
I had no hope that Rosenberger would send a model to me - a private person without an NDA. I tried it nevertheless - with no success. They did not even bother to respond to my email.
```

Funny rule: Say "NDE" and tell me what the first sound is you make. It is an "a" -> use "an".